settings.gradle

```diff
@@ -1,2 +1 @@
-rootProject.name = 'junit-pioneer-parent'
```

Comment:
I changed this back to leave it to #25 to do the multi-module build in one swoop.